cmake/sysbuild/suit.cmake

```diff
@@ -165,6 +165,11 @@ function(suit_create_package)
 
   suit_set_absolute_or_relative_path(${SB_CONFIG_SUIT_ENVELOPE_EDITABLE_TEMPLATES_LOCATION} ${PROJECT_BINARY_DIR} INPUT_TEMPLATES_DIRECTORY)
   set(ENVELOPE_SHALL_BE_SIGNED ${SB_CONFIG_SUIT_ENVELOPE_SIGN})
+  if(NOT DEFINED west_realpath)
+    # Twister build - ignore and store in the build folder!
+    set(SB_CONFIG_SUIT_ENVELOPE_EDITABLE_TEMPLATES_LOCATION "./")
+  endif()
+  sysbuild_get(ENVELOPE_SHALL_BE_SIGNED IMAGE ${DEFAULT_IMAGE} VAR CONFIG_SUIT_ENVELOPE_SIGN KCONFIG)
```

Comment:
Not for this PR, but how about copying the templates into `build/<sample-name>/...` directory? The flow would be as follows:
1. If template exists at `<sample-dir>`, then copy it to `build/<sample-name>/...` and use it to generate the envelope
2. If template does not exist at `<sample-dir>`, then copy it from default location into `<sample-dir>` and then do action from 1.

Reply:
I'll create a JIRA ticket to discuss that feature request

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-27662